pom.xml

```diff
@@ -476,7 +476,7 @@ under the License.
   <dependencies>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
`junit-jupiter-api` is enough - this artifact is really used in code

by the way I prefer use dependency in place where is used not like here for all modules - but it can be another change